py3status/modules/yandexdisk_status.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+Display Yandex.Disk status.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds (10s default)
+    format: prefix text for the Yandex.Disk status
+
+Valid status values include:
+    - Yandex.Disk isn't running!
+    - Error: daemon not started
+    - ...
```

Comment:
Sorry, maybe I wasn't clear.

The docstring for the module (the one we are in here) should have all the info.  stuff like 'Configuration parameters:'  __must__ be in it. This should be written in a style that users can understand, including people who have no idea what the module is.

README.md is generated by `py3status docstring update <module>`